Cargo.lock

```diff
@@ -2790,6 +2790,7 @@ dependencies = [
  "rstest",
  "rusqlite",
  "rust-embed",
+ "rust-ini",
```

Comment:
Curious what happened here.

Is this an artifact from @WindSoilder 's pluginification of `from ini`?

Reply:
hmmm, there's a rust-ini in the nu-command cargo.toml. Maybe it was missed? I'll remove it and see what happens.